.github/workflows/build.yml

```diff
@@ -11,8 +12,8 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - run: |
-          which vcpkg
-          type vcpkg
+          find / -name vcpkg-init.sh
+      - run: |
           vcpkg activate
           csolution list packs -m -s GetStarted.csolution.yml > packs.txt
```

Comment:
should we consider using cbuild here (once cmsis-toolbox >= 1.5.0) or do you prefer to do the step by step commands yourself?

`cbuild GetStarted.csolution.yml --packs`

will build all contexts.